changelog/shortenedMethodsEnabled.dd

```diff
@@ -0,0 +1,2 @@
+Shortened methods are now enabled by default, that is to say no preview flag
```

Comment:
1) Title should stay under 80 chars
2) Wording used (that is to say) is not professional
3) Add a description of the feature. When was it introduced ? Which DIP ? What does it do ?

Reply:
Why is it not professional?

You may mean legalese but the wording is completely fine in any professional context.

Reply:
will add something to changelog later, ran out of successful keypresses today.

Reply:
Suggestion for a better changelog entry:
```
Shortened method syntax is now enabled by default

Following the acceptance of [DIP 1043---Shortened Method Syntax](https://forum.dlang.org/thread/ahswpyijfidyorixgaqm@forum.dlang.org), you no longer need to use the `-preview=shortenedMethods` flag to use shortened function bodies:

---
int add(int x, int y) pure => x + y;

// equivalent full function body:
int add(int x, int y) pure { return x + y; }
---

The now obsolete preview flag will be deprecated in a future release.
```